### PR TITLE
fix standard-filters argument in cheetah

### PR DESCRIPTION
**Pull request by bernt-matthias**
fixes https://github.com/galaxyproject/tools-iuc/issues/4270

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by gregvonkuster** [comment]
Awesome - thanks for the quick fix!

**Comment by gregvonkuster** [comment]
This looks like it will now work, but is this the same tool that is in the TS here https://toolshed.g2.bx.psu.edu/view/devteam/freebayes/8a7a42541080?  

When installed from the TS, the tool only has this:
```
$ grep standard_filters *
freebayes.xml:                $standard_filters
```
But in the tools-iuc repo it has this:
```
$ grep standard_filters *
freebayes.xml:                        <param name="standard_filters" argument="--standard-filters" type="boolean" truevalue="-0" falsevalue="" checked="false"
```

**Comment by bernt-matthias** [comment]
Have you updated you master branch? It should look like this: https://github.com/galaxyproject/tools-iuc/blob/b63133897598f2881de7bad1f18fcd69d7d7e9df/tools/freebayes/freebayes.xml#L485

File: tools/freebayes/freebayes.xml [hunk 1]
```diff
@@ -181,7 +181,7 @@
 
         ##INPUT FILTERS
             #if str( $options_type.input_filters.input_filters_selector ) == "set":
-                $standard_filters
+                $options_type.input_filters.standard_filters
```
Comment (nsoranzo) [inline tools/freebayes/freebayes.xml:184]:
```suggestion
                ${options_type.input_filters.standard_filters}
```

Comment (bernt-matthias) [reply tools/freebayes/freebayes.xml:184]: Thanks for the suggestion @nsoranzo .. can you tell me what the curly braces are good for?

**Comment by gregvonkuster** [comment]
> Have you updated you master branch? It should look like this:
> 
> https://github.com/galaxyproject/tools-iuc/blob/b63133897598f2881de7bad1f18fcd69d7d7e9df/tools/freebayes/freebayes.xml#L485

Yes, that is in the tools-iuc repo, but the TS version must be older (or something?) than what is in the tools-iuc repo.

File: tools/freebayes/freebayes.xml (hunk 1 above)
Comment (nsoranzo) [reply tools/freebayes/freebayes.xml]: See https://cheetahtemplate.org/users_guide/language.html#placeholder-syntax-rules

They are optional and really only needed if you have something like `${var}string`, but they may improve readability. Same as in Bash, basically.
In this case my suggestion was mainly for consistency with the rest of the Cheetah code.

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**